### PR TITLE
update ocr2keepers to latest

### PR DESCRIPTION
**Pull request by infiloop2**
Syncing with https://github.com/smartcontractkit/ocr2keepers/pull/72

### CODE REVIEW

File: core/cmd/app.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func NewApp(client *Client) *cli.App {
 		},
 	}
 	app.Before = func(c *cli.Context) error {
-		if c.IsSet("config") || v2.EnvConfig != "" {
+		if c.IsSet("config") || v2.EnvConfig.Get() != "" {
```
Comment (infiloop2) [inline core/cmd/app.go:83]: TODO: this is an unrelated change in this PR and will be removed before merging.

File: core/scripts/chaincli/handler/report.go [hunk 2]
```diff
@@ -353,8 +353,7 @@ func (t *OCR2TransmitTx) SetStaticValues(elem *OCR2ReportDataElem) {
 	keys := []string{}
 	chkBlocks := []string{}
 	for _, u := range upkeeps {
-		parts := strings.Split(string(u.Key), "|")
-
+		parts := strings.Split(u.Key.String(), "|")
```
Comment (ferglor) [inline core/scripts/chaincli/handler/report.go:356]: I guess we can use `BlockKeyAndUpkeepID` now but this is fine for now

**Review by ferglor — APPROVED** [review]

**Review by begmaroman — APPROVED** [review]
